lib/shim/shim.js

```diff
@@ -471,8 +473,8 @@ Shim.prototype[symbols.unwrap] = unwrapAll
 /**
  * Entry point for executing a spec.
  *
- * @param nodule
- * @param spec
+ * @param {object|Function} nodule Class or module containing the function to wrap.
+ * @param {Spec} spec @see Spec
```

Comment:
`@see` is a top-level key like `@param`. To inline it, you'd do:
```suggestion
 * @param {Spec} spec {@link Spec}
```

But that is redundant because `@param {Spec}` _should_ link to "Spec".